src/Parser/HolyPriest/CPM_ABILITIES.js

```diff
@@ -15,7 +15,8 @@ const CPM_ABILITIES = [
     spell: SPELLS.PRAYER_OF_MENDING_CAST,
     name: 'Prayer of Mending',
     category: SPELL_CATEGORY.ROTATIONAL,
-    getCooldown: (haste, combatant) => (12 - (combatant.hasTalent(SPELLS.PIETY_TALENT.id) ? 2 : 0)) / (1 + haste),
+    getCooldown: (haste, combatant) => ((12 - (combatant.hasTalent(SPELLS.PIETY_TALENT.id) ? 2 : 0)) + 1.5) / (1 + haste), // +1.5 for base cast time
+    recommendedCastEfficiency: 0.70, // There's a lot of things that can potentially delay PoM: movement, heavy damage, etc. It is important to maintain high uptime but delaying a few GCDs is not bad.
```

Comment:
I guess I just wasn't sure the default value and how it generated minor/average/major thresholds off this value. I might have overexaggerated in my comment, but I'd say 70-80% (especially with the PoM CD fix) is fairly ideal for cast averages most of the time, but I guess I wouldn't want a "minor fix" at like 90% depending how it determines minor from this.